src/driver/dto/driver.dto.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @typedef {object} DriverDto
+ * @property {?string} id
+ * @property {?string} createdAt
+ * @property {?string} updatedAt
+ * @property {string} driverLicenceId
+ * @property {number} userId
+ */
+export const driverDto = {
+  type: 'object',
+  properties: {
+    id: {
+      type: 'number',
+    },
+    createdAt: {
+      type: 'string',
+      format: 'date',
+    },
+    updatedAt: {
+      type: 'string',
+      format: 'date',
+    },
+    driverLicenceId: {
+      type: 'string',
+      pattern: '^[A-Z]{2}\\d{7}$',
```

Comment:
If use regexp explain in comment what it does

Reply:
```js
// driver lisence format. e.g. first 2 letter A-Z, then 7 digits
pattern: '^[A-Z]{2}\\d{7}$',
```
I can give a wrong suggestion please correct

Reply:
no, this is correct, I just someway missed you comment during previous review